test/test-this-output.txt

```diff
@@ -0,0 +1,28 @@
+----------------------------------
```

Comment:
I'm not sure if these files should be commited. Maybe it is a good idea to have an example output, but I prefer to put it in the Wiki, not commit it. Please, take this into account.

Reply:
Fixed it.